oracles/src/domain/llm/generate_response_use_case.py

```diff
@@ -97,13 +97,13 @@ async def _generate_groq_with_params(chat: Chat) -> Optional[GroqChatCompletion]
 
 async def execute(model: str, chat: Chat) -> LLMResult:
     try:
-        if not chat.config or chat.prompt_type == PromptType.DEFAULT:
-            chat.prompt_type = PromptType.DEFAULT
-            response = await _generate(model=model, messages=chat.messages)
-        elif chat.prompt_type == PromptType.OPENAI:
+        if chat.prompt_type == PromptType.OPENAI:
             response = await _generate_openai_with_params(chat)
         elif chat.prompt_type == PromptType.GROQ:
             response = await _generate_groq_with_params(chat)
+        elif not chat.config or chat.prompt_type == PromptType.DEFAULT:
+            chat.prompt_type = PromptType.DEFAULT
+            response = await _generate(model=model, messages=chat.messages)
```

Comment:
The chat.config needs to be checked for none otherwise the `_generate_openai_with_params(chat)` will fail

Reply:
Reverted, I made a mistake